### ISSUE TITLE
Just cloned repository has modified files

### ISSUE THREAD

**Issue by ricpelo** [okamstudio/godot#2579]
1. `$ git clone https://github.com/okamstudio/godot.git`
2. `$ cd godot`
3. `$ git status` gives the following output:

````
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   core/io/aes256.cpp
	modified:   core/io/aes256.h
	modified:   drivers/etc1/rg_etc1.cpp
	modified:   drivers/etc1/rg_etc1.h
	modified:   drivers/nedmalloc/malloc.c.h
	modified:   drivers/nedmalloc/nedmalloc.cpp
	modified:   drivers/nedmalloc/nedmalloc.h
	modified:   drivers/openssl/register_openssl.cpp
	modified:   drivers/openssl/register_openssl.h
	modified:   drivers/rtaudio/RtAudio.cpp
	modified:   drivers/speex/config.h
	modified:   drivers/speex/lsp.h
	modified:   drivers/speex/speex_bind.cpp
	modified:   drivers/speex/speex_bind.h
	modified:   platform/flash/os_flash.h
	modified:   platform/winrt/app.cpp
	modified:   platform/winrt/app.h
	modified:   platform/winrt/detect.py
	modified:   platform/winrt/include/EGL/egl.h
	modified:   platform/winrt/include/EGL/eglext.h
	modified:   platform/winrt/include/EGL/eglplatform.h
	modified:   platform/winrt/include/GLES2/gl2.h
	modified:   platform/winrt/include/GLES2/gl2ext.h
	modified:   platform/winrt/include/GLES2/gl2platform.h
	modified:   platform/winrt/include/GLES3/gl3.h
	modified:   platform/winrt/include/GLES3/gl3ext.h
	modified:   platform/winrt/include/GLES3/gl3platform.h
	modified:   platform/winrt/include/GLSLANG/ShaderLang.h
	modified:   platform/winrt/include/GLSLANG/ShaderVars.h
	modified:   platform/winrt/include/KHR/khrplatform.h
	modified:   platform/winrt/include/angle_gl.h
	modified:   platform/winrt/include/angle_windowsstore.h
	modified:   servers/physics/space_sw.cpp
	modified:   servers/physics/space_sw.h
	modified:   tools/editor/plugins/path_editor_plugin.cpp
	modified:   tools/editor/spatial_editor_gizmos.cpp
	modified:   tools/editor/spatial_editor_gizmos.h
	modified:   tools/export/blender25/godot_export_manager.py

no changes added to commit (use "git add" and/or "git commit -a")
````

**Comment by akien-mga** [comment]
I think it's due to the new .gitattributes pushed in 3e6aaec which enforces LF (Unix-like) line endings.
I guess one of us (@reduz) should make a commit to confirm all those line ending changes; it might impact some of the existing PRs that depend on those files, but such is life.

Another solution is to revert the .gitattributes changes... but I think it's good for us to enforce a common line ending for all text files, regardless of the platform they are being edited on.

**Comment by ricpelo** [comment]
+1 to commit the LF changes ;).

**Comment by bojidar-bg** [comment]
Isn't this fixed by https://github.com/okamstudio/godot/commit/aad2bbdb6fb7c8217d7e75480b38e45f00cb3abd ?

**Comment by okamstudio** [comment]
likely fixed

On Fri, Oct 9, 2015 at 11:38 AM, Bojidar Marinov <notifications@github.com>
wrote:

> Isn't this fixed by aad2bbd
> <https://github.com/okamstudio/godot/commit/aad2bbdb6fb7c8217d7e75480b38e45f00cb3abd>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/okamstudio/godot/issues/2579#issuecomment-146890903>.
>



-- 

*OkamStudio*

**Comment by ricpelo** [comment]
Yes, fixed for me now ;).
